balancer/grpclb/grpclb.go

```diff
@@ -350,7 +350,7 @@ func (lb *lbBalancer) HandleResolvedAddrs(addrs []resolver.Address, err error) {
 
 	if lb.ccRemoteLB == nil {
 		if len(remoteBalancerAddrs) <= 0 {
-			grpclog.Errorf("grpclb: no remote balancer address is available, should never happen")
+			grpclog.Warningf("grpclb: no remote balancer address is available, should never happen")
```

Comment:
"should never happen" sounds like an internal assertion? I think error is fine in that case.

Reply:
Done